src/templates/default/reviews/review_change.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'dashboard_base.html' %}
+
+{% load i18n crispy_forms_tags %}
+
+{% block dashboard_tablist %}
+{% include '_includes/dashboard_tablist.html' with active='admin' %}
+{% endblock dashboard_tablist %}
+
+{% block main-content %}
+
+<h3 class="dashboard-header">
+	<span class="header-title">{% trans 'Change Review' %}</span>
+</h3>
+
+<div class="col-md-9">
+	<div class="dashboard-body">
+		<div class="panel panel-default">
+		<div class="panel-heading">{% trans 'Change Review' %}</div>
+			<ul class="list-group">
+				<li class="panel-title list-group-item">
+					pycontw-2020.proposals.creatable 
```

Comment:
No sure I remember correctly or ont: there should be a "slug" or something to represent "pycontw-2020". May be it is better for us to use a general slug rather than a year-specific prefix, so it will be more probable and maintainable in the upcoming years.

Reply:
https://github.com/pycontw/pycon.tw/blob/2fbd41bb3eb38d791fb152f9b7020432c770e48f/src/pycontw2016/settings/base.py#L301

This can be written as `{{ settings.CONFERENCE_DEFAULT_SLUG }}.proposals.creatable`, but I would instead favour [using a template tag](https://docs.djangoproject.com/en/3.0/howto/custom-template-tags/#writing-custom-template-tags) instead to automatically generate the repeating configuration key.

The ideal outcome would be something like

```html
<ul class="list-group">
  <li class="panel-title list-group-item">
    {% configuration_switch key='proposals.creatable' %}
  </li>
  <li class="panel-title list-group-item">
    {% configuration_switch key='proposals.editable' %}
  </li>
  <li class="panel-title list-group-item">
    {% configuration_switch key='proposals.withdrawable' %}
  </li>
  <li class="panel-title list-group-item">
    {% configuration_switch key='reviews.visible.to.submitters' %}
  </li>
  <li class="panel-title list-group-item">
    {% configuration_input key='reviews.stage' type='number' min=0 max=2 placeholder='int' required=True %}
  </li>
  <li class="panel-title list-group-item">
    {% configuration_input key='proposals.disable.after' type='datetime-local' step='1' placeholder='yyyy-mm-dd hh:mm:ss' %}
  </li>
</ul>
```